chpl/chpl-service/src/main/java/gov/healthit/chpl/builder/ComplaintsWorksheetBuilder.java

```diff
@@ -154,7 +154,7 @@ public Sheet buildWorksheet(final SurveillanceReportWorkbookWrapper workbook, fi
 
         Name complaintStatusTypeNamedCell = workbook.getWorkbook().createName();
         complaintStatusTypeNamedCell.setNameName("ComplaintStatusTypeList");
-        reference = "Lists!$F$1:$F$" + getNumberOfComplaintStatusTypes();
+        reference = "Lists!$F$1:$F$" + 2;
```

Comment:
Use a constant here instead of magic number 2. Even better make an array of the actual choices (open and closed) and get the length of the array.